config/jobs/ppc64le-cloud/all-in-one/allinone-postsubmit.yaml

```diff
@@ -13,15 +13,6 @@ postsubmits:
         serviceAccountName: build-img
         containers:
           - image: quay.io/powercloud/all-in-one:0.5
-            env:
```

Comment:
I don't see any preset added here for this job after removing the volumes!

Reply:
Missed adding. Will do in next patch.